modules/ducktests/tests/ignitetest/utils/ignite_test.py

```diff
@@ -27,6 +27,7 @@
 # globals:
 JFR_ENABLED = "jfr_enabled"
 IGNITE_TEST_CONTEXT_CLASS_KEY_NAME = "IgniteTestContext"
+SAFEPOINT_LOGS_DISABLED = "SafePointLogsDisabled"
```

Comment:
Please also change it *_enabled (with True by default). In all other variables we have enabled but not disabled.

Say "safepoint_log_enabled".

Reply:
Done

Reply:
just for record 

After all it was decided to have it False by default to not increase total logs size in results for tests which do not need it